examples/README.md

```diff
@@ -0,0 +1,164 @@
+more advanced than others. The following are some of the most prominent examples
+which you should take a look at.
+
+## Level: Beginner
+### Simple Square
+
+![square](../docs/assets/images/examples/square.gif)
+
+**Source Code**: [square.rs](square.rs)
+
+This is almost the simplest turtle program possible: it draws a non-centered
+square on the canvas by repeating the two basic `forward` and `right` methods.
+
+### Simple Circle
+
+![circle](../docs/assets/images/examples/circle.gif)
+
+**Source Code**: [circle.rs](circle.rs)
+
+The circle is usually the next step after simple geometric shapes like polygons
+because of its apparent simplicity. There is a catch though: a turtle may only
+turn in place or go in a straight line. So then we have to decompose the circle
+into small lines and turns in order to obtain a regular polygon with just a lot
+of sides so that it looks like a circle.
+
+### Dashed Circle
+
+![dashed_circle](../docs/assets/images/examples/dashed_circle.gif)
+
+**Source Code**: [dashed_circle.rs](dashed_circle.rs)
+
+This one is just a variant of the previous showcasing the ability to raise or
+lower the drawing pen using the `pen_up` and `pen_down` turtle methods.
+
+### Radioactive Triangles
+
+![three-triangles](../docs/assets/images/examples/three-triangles.gif)
+
+**Source Code**: [three-triangles.rs](three-triangles.rs)
+
+Shows how a simple operation like drawing an equilateral triangle can be repeated
+in order to get a more complex result.
+
+### Five Branch Star
+
+![empty_star](../docs/assets/images/examples/empty_star.gif)
+
+**Source Code**: [empty_star.rs](empty_star.rs)
+
+Simple five branch star with an empty middle that colors the background and the
+pen with a thicker pen size as well, using respectively the `set_background_color`,
+`set_pen_color` and `set_pen_size` turtle methods.
+
+## Level: Intermediate
+### Color-filled Squares Spiral
+
+![squares](../docs/assets/images/examples/squares.gif)
+
+**Source Code**: [squares.rs](squares.rs)
+
+This is a repetition of the square drawing, but with added colors and an inner
+colored fill using the `set_fill_color`, `begin_fill` and `end_fill` methods.
+
+### Isometric Nested Cubes
+
+![nestedcubes](../docs/assets/images/examples/nestedcubes.gif)
+
+**Source Code**: [nestedcubes.rs](nestedcubes.rs)
+
+This one is quite interesting: it draws a hexagonal spiral with a regularly
+fading color until it makes it go back to the original one, but it slowly
+reveals a form of visual illusion showing nested concentric three-dimensional
+cubes isometrically projected onto the two-dimensional space of the canvas.
+
+### Randomly-colored Tornado
+
+![coloredridge](../docs/assets/images/examples/coloredridge.gif)
+
+**Source Code**: [coloredridge.rs](coloredridge.rs)
+
+This color frenzy is also nice to look at: the regular shape is generated using
+a gaussian function setting the pen size.
+
+### Heart Emoji
```

Comment:
```suggestion
### Heart
```

The ❤️ emoji is actually quite different (depending on your platform), so I think this is fine just titled as "Heart". :)